### PR TITLE
Expose the head SEO function to be used in layouts

### PR DESCRIPTION
**Pull request by adrienbaron**
This PR exposes the SEO head hook function as `this.$nuxtI18nSeo` on components.

This allows to disable seo head mixin on all components (by setting `seo: false` in nuxt i18n options).
Then it's possible to do this in your app layout (or wherever):
```js
    head () {
      return this.$nuxtI18nSeo()
    },
```
This greatly improves performance as SEO links are not recomputed for every components but only for the layout itself.

This changes is not breaking as the default global registration with the same behaviour remains if `seo: true` in the configuration.

Resolves: #144

### CODE REVIEW

**Comment by paulgv** [comment]
Thanks @adrienbaron !
Could you also update the documentation to explain how to use the `nuxtI18nSeo()` method?

**Comment by adrienbaron** [comment]
@paulgv Done!
Btw thanks a lot to @manniL for his help on this, forgot to mention, and couldn't have done it without his help 👍

**Comment by manniL** [comment]
Happy we found a solution 🙊  (Also one that can be default at the next major version)

File: src/plugins/main.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import Vue from 'vue'
 import VueI18n from 'vue-i18n'
+import { nuxtI18nSeo } from './seo-head'
 
 Vue.use(VueI18n)
 
-export default async ({ app, route, store, req }) => {
+export default async ({ app, route, store, req }, inject) => {
```
Comment (manniL) [inline src/plugins/main.js]: we can omit that `inject` param now ☺️ @adrienbaron

**Comment by kevinmarrec** [comment]
@adrienbaron @manniL Last commit on `master` seems to be related to this PR, and tests are not passing anymore, any ideas ? (CircleCI build fails)

**Comment by manniL** [comment]
@kevinmarrec Should be just the vue-ssr id(`4eb03412:0`). Will fix

**Comment by manniL** [comment]
@kevinmarrec fixed

**Comment by manniL** [comment]
@harmnot There hasn't been a release yet

**Comment by harmnot** [comment]
I was asking this https://cmty.app/nuxt/nuxt-i18n/issues/c176

**Comment by harmnot** [comment]
then, I try to find out, and follow the instruction https://github.com/nuxt-community/nuxt-i18n/blob/master/docs/seo.md here and after that , show the $nuxtI18nSeo is not a function

**Comment by paulgv** [comment]
Sorry for the delay, this wasn't released until recently in version `5.5.0`: https://github.com/nuxt-community/nuxt-i18n/blob/master/CHANGELOG.md#550-2019-01-14

**Pull request merged**